doc/man/scons.xml

```diff
@@ -4843,36 +4945,53 @@ opt.AddVariables(
   <varlistentry id="v-Update">
   <term><replaceable>vars</replaceable>.<function>Update</function>(<parameter>env, [args]</parameter>)</term>
   <listitem>
-<para>Update a &consenv;
+<para>Process the input sources recorded
+when the &Variables; object was initialized
+and update
 <parameter>env</parameter>
 with the customized &consvars;.
-Any specified variables that are not
-configured for the Variables object
-will be saved and may be
-retrieved using the
+The names of any variables in the input sources that are not
+configured in the &Variables; object
+are recorded and may be retrieved using the
 <link linkend='v-UnknownVariables'>&UnknownVariables;</link>
 method.</para>
 
-<para>Normally this method is not called directly,
-but rather invoked indirectly by passing the Variables object to
+<para>
+If the optional
+<parameter>args</parameter>
+argument is provided, it is a dictionary of variables
+to use in place of the one saved when
+<link linkend='v-Variables'>&Variables;</link>
+was called.
+</para>
+
+<para>Normally, &Update; is not called directly,
+but rather invoked indirectly by passing the &Variables; object to
 the &f-link-Environment; function:</para>
 
 <programlisting language="python">
-env = Environment(variables=vars)
+env = Environment(..., variables=vars)
 </programlisting>
 
+<para>
+</para>
```

Comment:
why empty para?

Reply:
Remnant of some editing exercise, I guess. Dropped.